ila.py

```
@@ -1,6 +1,11 @@
 from ncclient import manager
 import xmltodict
 from utils import *
```

Ruff (0.12.2)

`from utils import *` used; unable to detect undefined names

(F403)